source/_components/notify.amazon_chime

```diff
@@ -0,0 +1,42 @@
+# ---
+layout: page
+title: "Amazon Chime"
+description: "Instructions on how to send messages to Amazon Chime from Home Assistant."
+date: 2019-03-03 00:44
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Notifications
+ha_release: "TBD"
+---
```

Comment:
Missing the `ha_iot_class`

Reply:
Is this required for a notification component?